### ISSUE TITLE
Accessibility issue | Stat panel | Keyboard navigation | Screen reader

### ISSUE THREAD

**Issue by Darkavid** [grafana/grafana#84242]
**Steps to reproduce**:
1. Create a dashboard
2. Add a new panel
3. Set the visualization to 'Stat'
4. Add some basic query to get some data
5. Save the panel
6. Add another panel (repeat steps 2#, 3#, 4# and 5#)
7. Make sure the focus is "before" all panels
8. Use keyboard (e.g. via Tab key) to navigate and move focus between panels

**Actual Result**:
I can navigate between Stat panels but I cannot move focus between the inner stat blocks (which present data) as they miss tab-index.

**Expected Result**:
Via the keyboard, I should be able to navigate also between the inner stat blocks as well. 

**Rationale**:
It is common that a Stat panel contains many inner stat blocks. Having no possibility to move the focus individually between those blocks breaks the accessibility while using a screen-reader. Even if users are curious only about some particular information in a Stat panel, they need to listen all stat information presented in all inner stat blocks and if they missed some, they need to listen all contents of the panel again for no gain at all.

**Relevant WCAG Criteria:**
[2.1.1 WCAG Criterion](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

**Environment**:
- Grafana version: Grafana v10.2.3 (1e84fede54)
- Data source type & version: Any (irrelevant)
- User OS & Browser: Windows 11 & Google Chrome (Version 122.0.6261.112)
